coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -57,11 +58,21 @@ def apply(self, result, original_file_dict, file_diff_dict, language: str,
         filename = source_range.file
 
         ignore_diff = Diff(original_file_dict[filename])
-        ignore_diff.change_line(
-            source_range.start.line,
-            original_file_dict[filename][source_range.start.line-1],
-            original_file_dict[filename][source_range.start.line-1].rstrip() +
-            '  ' + ignore_comment)
+
+        original_line = str(original_file_dict[filename][
+                        source_range.start.line-1].rstrip())
+
+        # Add the new ignore using 'and' with the previous ignore.
+        if re.search(r'Ignore.+Bear', original_line):
```

Comment:
From previous comment:
> Do we require that ignores need to end with `Bear`? I think it depends on the origin^^

Reply:
@Makman2 doesn't every bear has ``Bear`` at the end? If not, then I have to search only for ``Ignore`` in the line. Please convey.

Reply:
I have searched https://coala.io/#/languages , every bear has a ``Bear`` with it. If I am missing something, do let me know :wink: